metrics/metrics.go

```diff
@@ -163,11 +161,15 @@ func CollectProcessMetrics(refresh time.Duration) {
 		diskWriteBytesCounter = GetOrRegisterCounter("system/disk/writebytes", DefaultRegistry)
 	)
 
+	// Avoid divide-by-zero the first time through the loop.
+	lastCollectionTime := time.Now().Add(-refresh)
```

Comment:
It's very wonky to me. You are adding a negative `refresh` nano seconds.

Reply:
I know that looks weird.  There is a `Sub` function, but it only subtracts a Time value, not a Duration.  According to the documentation, that's how you subtract a Duration: `To compute t-d for a duration d, use t.Add(-d).`

https://pkg.go.dev/time#Time.Sub